libraries/AP_BattMonitor/AP_BattMonitor_SMBus_Maxell.cpp

```diff
@@ -95,6 +96,12 @@ void AP_BattMonitor_SMBus_Maxell::timer()
         _state.current_amps = -(float)((int16_t)data) / 1000.0f;
         _state.last_time_micros = tnow;
     }
+
+    // read temperature
+    if (read_word(BATTMONITOR_SMBUS_MAXELL_TEMP, data)) {
+        _state.has_temperature = true;
```

Comment:
Yeah. It's possible this is just a bus error, but we should probably flag it anyways, otherwise we will have a bad stale temperature without knowing it....